python/pylibcugraph/pylibcugraph/uniform_neighbor_sample.pyx

```diff
@@ -212,7 +249,18 @@ def uniform_neighbor_sample(ResourceHandle resource_handle,
 
     cdef cugraph_rng_state_t* rng_state_ptr = \
         cg_rng_state.rng_state_ptr
-    error_code = cugraph_uniform_neighbor_sample_with_edge_properties(
+
+    cdef cugraph_sampling_options_t* sampling_options
+    error_code = cugraph_sampling_options_create(&sampling_options, &error_ptr)
+    assert_success(error_code, error_ptr, "cugraph_sampling_options_create")
+
+    cugraph_sampling_set_with_replacement(sampling_options, with_replacement)
+    cugraph_sampling_set_return_hops(sampling_options, <bool_t>True)
```

Comment:
Why is this one hardcoded to True and not part of the python and PLC API?

Reply:
fixed